src/librustc_mir/interpret/eval_context.rs

```diff
@@ -38,9 +38,6 @@ pub struct InterpCx<'mir, 'tcx, M: Machine<'mir, 'tcx>> {
     /// The virtual memory system.
     pub memory: Memory<'mir, 'tcx, M>,
 
-    /// The virtual call stack.
-    pub(crate) stack: Vec<Frame<'mir, 'tcx, M::PointerTag, M::FrameExtra>>,
```

Comment:
Please leave a comment somewhere here that the stack is provided through the `Machine` trait. It seems somewhat surprising that the global machine state would not contain a call stack. ;)